wwwbasic.js

```diff
@@ -1906,7 +1906,8 @@
           Skip('goto');
           var name = tok;
           Next();
-          // TODO: Implement.
+          curop += 'ip = labels["' + name + '"];\n';
```

Comment:
So this is down where it parses ON x GOTO a,b,c,d
What you had there would work for GOTO.
What's needed is actually parse a bunch of labels into a list and generated a lookup based on the int value.
I've filed an issue to track:
https://github.com/google/wwwbasic/issues/25

Reply:
Ok well I'll try to stick to what I know when it comes to pull requests then 🤷‍♂️ Maybe I'll try to throw together some docs next...